backend/services/customer-ldap/README.md

```
@@ -0,0 +1,454 @@
+
+All requests require either:
+- **Authorization header**: `Bearer <omnistrate-jwt-token>`
+- **Session cookie**: `ldap-session` (automatically set after first successful request)
+
+### Endpoints
+
+#### List Users
+
+```
+GET /instances/:instanceId/users?subscriptionId=<subscription-id>
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)